src/less/techne-patterns/_product-thumbnail.less

```diff
@@ -114,11 +120,22 @@
     .product-price {
         font-size: 15px;
         margin-bottom: 10px;
-    }
 
-    .product-price-discount {
-        .text-danger;
-        display: inline-block;
+        .product-price-element { 
+            display: block;
```

Comment:
This breaks the existing single price display. The discounted price will always display below.

**Range**
```
47.48 - 49.48
40.20 - 42.48 USD
```

**One price**
```
47.48
40.20 USD
```